build_tools/github/test_windows_wheels.sh

```diff
@@ -18,6 +18,10 @@ else
                --name minimal_windows \
                -d -ti --rm scikit-learn/minimal-windows
 
+    docker exec minimal_windows powershell -NoProfile -Command "$env:SKLEARN_SKIP_NETWORK_TESTS"
+    docker exec minimal_windows powershell -NoProfile -Command "$env:OMP_NUM_THREADS"
+    docker exec minimal_windows powershell -NoProfile -Command "$env:OPENBLAS_NUM_THREADS"
```

Comment:
You probably need `Write-Output` or something similar to display env variables in powershell.

Reply:
As you may notice, I am not particularly familiar with `powershell` :sweat_smile:.